File: Php/core/PageModel.php
namespace core{

	use app\User\UserNavigationModel;
	use core\Analytics\GoogleAnalyticsModel;
	use entities\Homegame;
	use Domain\Classes\User;

	class PageModel {

		public $validationErrors;
		public $userNavigationModel;
		public $homegameNavigationModel;
		public $googleAnalyticsModel;

		public function __construct(User $user = null){
			$this->userNavigationModel = new UserNavigationModel($user);
			$this->googleAnalyticsModel = new GoogleAnalyticsModel();
		}

		public function setValidationErrors(array $errors){
			$this->validationErrors = $errors;
		}

	}

}

File: Web/Models/PageModel.cs
using System.Collections.Generic;
using Core.Classes;
using Web.Models;

namespace core{

	public class PageModel {

	    public List<string> ValidationErrors { get; set; }
        //todo:Flytta UserNavigationModel
	    //public UserNavigationModel UserNavModel { get; set; }
	    public GoogleAnalyticsModel GoogleAnalyticsModel { get; set; }

	    public PageModel(User user)
	    {
	        //UserNavModel = new UserNavigationModel(user);
			GoogleAnalyticsModel = new GoogleAnalyticsModel();
	    }

		public void SetValidationErrors(List<string> errors){
			ValidationErrors = errors;
		}

	}

}